apps/condo/domains/organization/integrations/sbbol/SbbolCredentials.js

```diff
@@ -0,0 +1,95 @@
+const querystring = require('querystring')
+const conf = require('@core/config')
+const { getOrganizationAccessToken } = require('./accessToken')
+const { SbbolRequestApi } = require('./SbbolRequestApi')
+const { Organization, TokenSet } = require('@condo/domains/organization/utils/serverSchema')
+const { SBBOL_IMPORT_NAME } = require('./common')
+const dayjs = require('dayjs')
+const { getSchemaCtx } = require('@core/keystone/schema')
+
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
+
+class SbbolCredentials {
```

Comment:
SbbolCredentialsHelper?

Reply:
Ideally, I would get rid of classes in this case, because there will be no instances with some lifecycle.
It was made for consistency with other utility classes.